config/cmake/modules/MfemCmakeUtilities.cmake

```diff
@@ -900,6 +921,45 @@ function(mfem_export_mk_files)
   string(STRIP
          "${cxx_std_flag} ${CMAKE_CXX_FLAGS_${BUILD_TYPE}} ${CMAKE_CXX_FLAGS}"
          MFEM_CXXFLAGS)
+  if (MFEM_USE_CUDA)
+    set(MFEM_CXXFLAGS "${MFEM_CXXFLAGS} ${CMAKE_CUDA_FLAGS}")
+    if (CMAKE_CUDA_COMPILER MATCHES "nvcc$")
+      # set(MFEM_CXXFLAGS "-isystem ${CUDAToolkit_LIBRARY_ROOT} ${MFEM_CXXFLAGS}")
```

Comment:
Is this commented out code left here on purpose?  If so, perhaps an inline comment as to why is warranted?